RRemoteConfig/ConfigModel.swift

```diff
@@ -29,11 +34,30 @@ internal struct KeyModel: Decodable, Parsable {
     let createdAt: String
 
     init?(data: Data) {
-        guard let dictionary = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String] else {
+        guard let dictionary = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String],
+            let id = dictionary["id"],
+            let key = dictionary["key"],
+            let createdAt = dictionary["createdAt"] else {
             return nil
         }
-        self.id = dictionary["id"] ?? ""
-        self.key = dictionary["key"] ?? ""
-        self.createdAt = dictionary["createdAt"] ?? ""
+        self.id = id
+        self.key = key
+        self.createdAt = createdAt
+    }
+}
+
+extension Data {
```

Comment:
You can declare this extension as private if it's only used here

Reply:
Yep, I'll do that. 
I will leave it until a follow-up model refactoring PR (already in progress).